frontend/app/routes/_public+/apply+/$id+/confirm.tsx

```diff
@@ -77,22 +77,47 @@ export default function ApplyFlowConfirm() {
   const { userInfo, spouseInfo, preferredLanguage, homeAddressInfo, mailingAddressInfo, dentalInsurance } = useLoaderData<typeof loader>();
   const { i18n, t } = useTranslation(i18nNamespaces);
 
-  const MSCALink = <InlineLink to={`https://www.canada.ca/${i18n.language === 'en' ? 'en/employment-social-development/services/my-account.html' : 'fr/emploi-developpement-social/services/mon-dossier.html'}`} />;
-  const dentalContactUsLink = <InlineLink to={`https://www.canada.ca/${i18n.language === 'en' ? 'en/services/benefits/dental/dental-care-plan/contact.html' : 'fr/services/prestations/dentaire/regime-soins-dentaires/contactez.html'}`} />;
+  const mscaLink = <InlineLink to={t('apply:confirm.msca-link')} />;
```

Comment:
We can remove the namespace when specifying the i18n key within the `t` function because we specified the `i18nNamespace` as `apply` within the `handle` function.

```suggestion
  const mscaLink = <InlineLink to={t('confirm.msca-link')} />;
```

This same comment can be applied throughout this page.